src/modules/roc_core/target_posix/roc_core/colors.cpp

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright (c) 2019 Roc authors
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+#include <stdio.h>
+#include <string.h>
+#include <unistd.h>
+
+#include "roc_core/colors.h"
+
+namespace roc {
+namespace core {
+
+//! ANSI Color Codes.
+#define COLOR_NONE       ""
+#define COLOR_START      "\033["
+#define COLOR_END        "m"
+#define COLOR_RESET      COLOR_START COLOR_END
```

Comment:
It seems that a [valid sequence](https://en.wikipedia.org/wiki/ANSI_escape_code) for resetting all attributes to their defaults is `\033[0m`.

Reply:
Hm, it seems that `\033[m` also works for me.